util/loadtrace.py

```
@@ -0,0 +1,112 @@
+    replaystr_idx = col_name.index('ReplayStr')
+    rows = cur.fetchall()
+    load_map = {}
+
+    def statistics(data, total):
+        pos = data[atfetch_idx: atcommit_idx + 1]
+        if pos[-1] == 0:
+            return total
+        disp_idx = 3
+        if pos[disp_idx] == 0:
+            pos[disp_idx] = pos[disp_idx - 1]
+        pos_diff = [j - i for i, j in zip(pos[:-1], pos[1:])]
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)